src/lib/types/substrate.ts

```diff
@@ -0,0 +1,4 @@
+export type Polkadot = {
+  api: any
```

Comment:
Is there anyway we can get rid of the `any` and type it? The fewer `any`s now will help with future work.

Reply:
updated to use the correct types

Reply:
Nice